projects/crypto-currency/iluminados/api/src/routes/index.js

```diff
@@ -3,9 +3,9 @@ import cors from 'cors'
 import dotenv from 'dotenv'
 import market from './modules/market.js'
 import feeds from './modules/feeds.js'
-import coinOfTheDay from './modules/coin-of-the-day.js'
+import CoinOfTheDay from './modules/coinoftheday.js'
 import register from './modules/register.js'
-import influencers from './modules/influencers.js'
+import Influencer from './modules/influencer.js'
```

Comment:
Seguindo o padrão dos outros imports, não deveria começar com minúsculo? `Influencer `-> `influencer`
```suggestion
import influencer from './modules/influencer.js'
```